apps/prairielearn/src/ee/lib/ai-grading/ai-grading-stats.ts

```diff
@@ -167,7 +167,7 @@ export async function calculateAiGradingStats(
   return stats;
 }
 
-async function selectGradingJobsInfo<T extends { id: string }>(
+export async function selectGradingJobsInfo<T extends { id: string }>(
```

Comment:
Bonus points for a JSDoc comment explaining what this does (selects the most recent `AI` and `Manual` grading jobs for each instance question).